src/pybind/mgr/dashboard/HACKING.rst

```diff
@@ -237,6 +237,8 @@ How to extract messages from source code?
 To extract the I18N messages from the templates and the TypeScript files just
 run the following command in ``src/pybind/mgr/dashboard/frontend``:
```

Comment:
It's probably simpler to [simply add another colon](http://docutils.sourceforge.net/docs/ref/rst/restructuredtext.html#literal-blocks) in line 238 instead of adding a `.. code-block::` directive for a single line:
```suggestion
run the following command in ``src/pybind/mgr/dashboard/frontend``::
```

Reply:
done